bundle/config/root.go

```diff
@@ -320,7 +320,7 @@ func (r *Root) MergeTargetOverrides(name string) error {
 		"presets",
 	} {
 		if root, err = mergeField(root, target, f); err != nil {
-			return err
+			return fmt.Errorf("failed to merge target=%s field=%s: %w", name, f, err)
```

Comment:
this and above error remain uncovered, I don't know an easy way to trigger it, but it is still worth having context in the error IMO.

Reply:
I believe it is unreachable code because the configuration is normalized before merging. That means the types match up and merging cannot fail. Still good to include the context in case this assumption changes.